drivers/hmis/app/models/hmis/filter/household_filter.rb

```diff
@@ -23,6 +24,7 @@ def with_statuses(scope)
         return scope if input.status.sort == ['ACTIVE', 'INCOMPLETE', 'EXITED'].sort
```

Comment:
Is this return like bumping us out of the the rest of the filter or something...? I'm still seeing the odd bug when all 3 statuses are selected.

Reply:
Per slack convo, this is a latent bug in the filters. It's fixed in https://github.com/greenriver/hmis-warehouse/pull/3622